### PR TITLE
Update nybb url.

### PR DESCRIPTION
**Pull request by micahcochran**
Change download_nybb() to return shapefile name and path within the zip file.   Fix for issue #295   A few changes to make the code a little more maintainable.

### CODE REVIEW

File: tests/util.py [hunk 1]
```diff
@@ -36,16 +37,26 @@ class OperationalError(Exception):
...
+            response = urlopen('http://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/{0}'.format(filename))
             f.write(response.read())
-    return full_path_name
+    
+    shp_zip_path = None
+    zf = zipfile.ZipFile(full_path_name, 'r')
+    # finds path name in zip file
+    for zip_filename_path in zf.namelist():
+        if zip_filename_path.endswith('nybb.shp'):
+            break
```
Comment (jorisvandenbossche) [inline tests/util.py]: Is this needed? The file is just called 'nybb.shp', or is it to be more robust to name changes?

**Comment by jorisvandenbossche** [comment]
Thanks a lot! This seems indeed a good solution for now.

File: tests/util.py (hunk 1 above)
Comment (micahcochran) [inline tests/util.py]: I tried this out.
```python
>>> zf.extract('nybb.shp', '.')
KeyError: "There is no item named 'nybb.shp' in the archive"
```
You have to include the path within the zip file `zf.extract('nybb_16a/nybb.shp', '.')`,  which has changed from the 2014 version and would presumably change in future versions.

This revision is an attempt to make it so only one line would to change one line to migrate nybb dataset from year to year.

**Comment by jorisvandenbossche** [comment]
OK, merging, so at least the tests don't fail anymore. 
We can always fine tune later

cc @kjordahl

**Comment by micahcochran** [comment]
Thanks.

**Pull request merged**